jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -387,7 +387,7 @@ nomad:
 
 [#gcp]
 ==== GCP
-The values for this section were created <<gcp-autoscaler,here>>
+You created these values in the <<gcp-service-account,Nomad Autoscaler Configuration section>>.
```

Comment:
Same comment as above, I believe the # is still needed in the URL for section links.
`<<#gcp-service-account,Nomad Autoscaler Configuration section>>`